### PR TITLE
Update templates

### PR DESCRIPTION
**Pull request by sven1103**
This PR makes some simple updates to match the current requirements.

### CODE REVIEW

File: qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy.life.qbic/example/Example.groovy [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package life.qbic.example
```
Comment (KochTobi) [inline qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy.life.qbic/example/Example.groovy:1]: The folder structure of the project is wrong. There should not be one folder named `groovy.life.qbic` but three folders/packages `gorrvy` `life` and `qbic`

Comment (sven1103) [reply qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy.life.qbic/example/Example.groovy:1]: damn, that was the IDE formatting it like this ...

Comment (KochTobi) [reply qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy.life.qbic/example/Example.groovy:1]: > If you want to create a number of nested directories, specify the directory names separated with slashes.

**Comment by KochTobi** [comment]
The ServiceIntegrationTests still try to use Junit. Should this be changed to Spock also?

**Comment by sven1103** [comment]
@KochTobi If you could rewrite it and open a new PR?

File: qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy/life/qbic/example/Example.groovy [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package life.qbic.example
```
Comment (Zethson) [inline qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy/life/qbic/example/Example.groovy:1]: That's a groovy file inside a Java template.

lib_java has no groovy support (yet!).

**Review by Zethson — CHANGES_REQUESTED** [review]

File: qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy/life/qbic/example/Example.groovy (hunk 2 above)
Comment (sven1103) [reply qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy/life/qbic/example/Example.groovy:1]: sure it has, via the parent pom ;)

Comment (Zethson) [reply qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy/life/qbic/example/Example.groovy:1]: Ahh, it's part of the parent pom.

In my (old) QBiC projects I included it manually :)

Still, for consistency I would prefer a .java here, but the choice is yours :)

**Review by Zethson — APPROVED** [review]
LGTM besides .groovy vs .java file name

File: qube/create/templates/portlet/portlet_groovy/{{ cookiecutter.project_slug }}/pom.xml [hunk 3]
```diff
@@ -28,7 +28,7 @@
 			</snapshots>
 			<id>nexus-snapshots</id>
 			<name>QBiC Snapshots</name>
-			<url>https://qbic-repo.am10.uni-tuebingen.de/repository/maven-snapshots</url>
+			<url>http://qbic-repo.qbic.uni-tuebingen.de:8081/repository/maven-snapshots</url>
```
Comment (KochTobi) [inline qube/create/templates/portlet/portlet_groovy/{{ cookiecutter.project_slug }}/pom.xml:31]: Do we need the port here?
Did this change from `https:` to `http`?

File: qube/create/templates/portlet/portlet_groovy/{{ cookiecutter.project_slug }}/pom.xml [hunk 4]
```diff
@@ -41,44 +41,39 @@
 			</snapshots>
 			<id>nexus-releases</id>
 			<name>QBiC Releases</name>
-			<url>https://qbic-repo.am10.uni-tuebingen.de/repository/maven-releases</url>
+			<url>http://qbic-repo.qbic.uni-tuebingen.de:8081/repository/maven-releases</url>
```
Comment (KochTobi) [inline qube/create/templates/portlet/portlet_groovy/{{ cookiecutter.project_slug }}/pom.xml:44]: Do we need the port here?
Did this change from `https:` to `http`?

File: qube/create/templates/service/service_java/{{ cookiecutter.project_slug }}/pom.xml [hunk 5]
```diff
@@ -29,7 +28,7 @@
 			</snapshots>
 			<id>nexus-snapshots</id>
 			<name>QBiC Snapshots</name>
-			<url>https://qbic-repo.am10.uni-tuebingen.de/repository/maven-snapshots</url>
+			<url>http://qbic-repo.qbic.uni-tuebingen.de:8081/repository/maven-snapshots</url>
```
Comment (KochTobi) [inline qube/create/templates/service/service_java/{{ cookiecutter.project_slug }}/pom.xml:31]: Do we need the port here?
Did this change from `https:` to `http`?

File: qube/create/templates/service/service_java/{{ cookiecutter.project_slug }}/pom.xml [hunk 6]
```diff
@@ -42,18 +41,10 @@
 			</snapshots>
 			<id>nexus-releases</id>
 			<name>QBiC Releases</name>
-			<url>https://qbic-repo.am10.uni-tuebingen.de/repository/maven-releases</url>
+			<url>http://qbic-repo.qbic.uni-tuebingen.de:8081/repository/maven-releases</url>
```
Comment (KochTobi) [inline qube/create/templates/service/service_java/{{ cookiecutter.project_slug }}/pom.xml:44]: Do we need the port here?
Did this change from `https:` to `http`?

File: qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/pom.xml [hunk 7]
```diff
@@ -29,7 +28,7 @@
 			</snapshots>
 			<id>nexus-snapshots</id>
 			<name>QBiC Snapshots</name>
-			<url>https://qbic-repo.am10.uni-tuebingen.de/repository/maven-snapshots</url>
+			<url>http://qbic-repo.qbic.uni-tuebingen.de:8081/repository/maven-snapshots</url>
```
Comment (KochTobi) [inline qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/pom.xml:31]: Do we need the port here?
Did this change from `https:` to `http`?

**Comment by KochTobi** [comment]
The portlet folder structure is also not correct. There is a folder with the name `life.qbic.portal.portlet`

File: qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy/life/qbic/example/Example.groovy (hunk 2 above)
Comment (KochTobi) [reply qube/create/templates/lib/lib_java/{{ cookiecutter.project_slug }}/src/main/groovy/life/qbic/example/Example.groovy:1]: my IDE does not recognize that groovy is allowed/ included in the projects.

**Pull request merged**